### PR TITLE
document locating c8 credentials

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Closes https://github.com/camunda/camunda-docs/issues/3267.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [x] This is already available but undocumented.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
**Will backport upon approval**.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineerring DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer.

### CODE REVIEW

File: src/pages/contact.md [hunk 1]
```diff
@@ -29,3 +29,18 @@ There are a few different channels you can reach us based on your needs:
...
 - For sales inquiries, information about Camunda 8 performance and benchmarking, or anything not listed above, use our [Contact Us](https://camunda.com/contact/) form.
+
+## Locating Camunda 8 credentials
+
+Need assistance locating your Camunda 8 credentials? You can obtain these credentials from Camunda by submitting a **Help Request**. To do this, take the following steps:
+
+1. Log in to [Jira](https://jira.camunda.com/secure/Dashboard.jspa).
+2. Click **Create** in the navigation bar at the top of the page. This launches a **Create Issue** pop-up.
+3. In the **Issue Type** field, select **Help Request**.
+4. In the **Help Request Type** field, click the option that reads **I need the credentials for downloading Camunda 7**.
```
Comment (christinaausley) [inline src/pages/contact.md:40]: This was my biggest question, and what leads me to think is this the correct guidance?

The field is labeled **I need the credentials for downloading Camunda 7**, but should this just read **I need the credentials for downloading Camunda**, or should we have a separate field for C8?

**Review by akeller — COMMENTED** [review]
👍 I like the changes here, but I'd like to see the help request types clarified on the UI itself so it's clear C8 customers can use this as well.

**Review by wollefitz — COMMENTED** [review]
In my opinion "Camunda 8 credentials" is a confusing term and could be easily confused with the upcoming changes in https://github.com/camunda/product-hub/issues/2286. Technically, what users require are credentials for accessing the Camunda Docker Registry (ie. our Harbor instance at https://registry.camunda.cloud), they are not necessarily tied to Camunda 8.

Heads-up: Starting with `8.6.0-alpha2` (ie. **not** older versions), Web Modeler's Docker images will be publicly available at Docker Hub, just like the other components (see https://github.com/camunda/product-hub/issues/2139). At this point, we can remove this from the `next` docs again.

**Comment by akeller** [comment]
> In my opinion "Camunda 8 credentials" is a confusing term and could be easily confused with the upcoming changes in [camunda/product-hub#2286](https://github.com/camunda/product-hub/issues/2286). Technically, what users require are credentials for accessing the Camunda Docker Registry (ie. our Harbor instance at https://registry.camunda.cloud), they are not necessarily tied to Camunda 8.
> 
> Heads-up: Starting with `8.6.0-alpha2` (ie. **not** older versions), Web Modeler's Docker images will be publicly available at Docker Hub, just like the other components (see [camunda/product-hub#2139](https://github.com/camunda/product-hub/issues/2139)). At this point, we can remove this from the `next` docs again.

@wollefitz, is support aware of this? Since DevEx doesn't own the Support portal page (screenshot in this PR), we will adopt their language, which sounds like it could be misleading now and in the future. Can you start a thread with Support and pull in @christinaausley?

**Comment by wollefitz** [comment]
> @wollefitz, is support aware of this? Since DevEx doesn't own the Support portal page (screenshot in this PR), we will adopt their language, which sounds like it could be misleading now and in the future.

Just had a look at the screenshot again - calling this "Camunda 8 download credentials" to be aligned with the support portal would already reduce the confusion. I'd not expect this being a problem after `8.6-alpha2` as all components will be publicly available and no download credentials will be required anymore. Note that this is just my personal opinion, I'm not the DRI of https://github.com/camunda/product-hub/issues/2286.

File: src/pages/contact.md (hunk 1 above)
Comment (valeriaportolesicamunda) [reply src/pages/contact.md:40]: At the moment the field is labeled "_I need the credentials for downloading Camunda_". The documentation on Camunda 7 is not updated.

Comment (valeriaportolesicamunda) [reply src/pages/contact.md:40]: This is how it looks from customer perspective:
credentials

Comment (christinaausley) [reply src/pages/contact.md:40]: Cool -- I will update!

**Review by valeriaportolesicamunda — APPROVED** [review]
It looks fine!

**Review by akeller — APPROVED** [review]
@christinaausley and I discussed this in our 1:1, and I think the PR is good to merge. There are some follow ups including updating the images on https://docs.camunda.org/enterprise/support/, but that's outside the scope of this PR.

Approved 👍

**Pull request merged**